charts/cronjob/templates/cronjob.yaml

```diff
@@ -0,0 +1,137 @@
+{{- $top := . }}
+{{- range .Values.cronjobs }}
+---
+apiVersion: batch/v1beta1
```

Comment:
Nit: switch to `batch/v1`?

Reply:
> Nit: should we switch to `batch/v1`?

`batch/v1` is not stable until Kubernets 1.21. Since we're on 1.19 we have to remain on `batch/v1beta1`. :(

Reply:
Resolved using the common chart update.